src/libs/elektra/hooks.c

```diff
@@ -81,6 +101,64 @@ static int initHooksSpec (KDB * kdb, Plugin * plugin, Key * errorKey)
 	return 0;
 }
 
+static int initHooksSendNotifications (KDB * kdb, const KeySet * config, KeySet * modules, const KeySet * contract, Key * errorKey)
+{
+	struct _SendNotificationHook * lastHook = kdb->hooks.sendNotification = NULL;
```

Comment:
```suggestion
	struct _SendNotificationHook * lastHook = kdb->hooks.sendNotification;
```

Why do we need to set it `NULL` here? Isn't this function called only after a `freeHooks`?

Reply:
Yes, this isn't actually needed.